### PR TITLE
Add a codeowners file

### PR DESCRIPTION
**Pull request by FlorentClarret**
Add a codeowners file

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+/pgbouncer                          @DataDog/agent-integrations
+/powerdns_recursor                  @DataDog/agent-integrations
+/varnish                            @DataDog/agent-integrations
+/voltdb                             @DataDog/agent-integrations
+
+# Database monitoring
+/sqlserver                          @DataDog/database-monitoring
+
+# Infrastructure integrations
+/snmp                               @DataDog/network-device-monitoring
```
Comment (FlorentClarret) [inline .github/CODEOWNERS:22]:
```suggestion
/snmp                               @DataDog/Network Device Monitoring
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+/pgbouncer                          @DataDog/agent-integrations
+/powerdns_recursor                  @DataDog/agent-integrations
+/varnish                            @DataDog/agent-integrations
+/voltdb                             @DataDog/agent-integrations
+
+# Database monitoring
+/sqlserver                          @DataDog/database-monitoring
+
+# Infrastructure integrations
+/snmp                               @DataDog/Network Device Monitoring
```
Comment (FlorentClarret) [inline .github/CODEOWNERS:22]:
```suggestion
/snmp                               @DataDog/infrastructure-integrations
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+/pgbouncer                          @DataDog/agent-integrations
+/powerdns_recursor                  @DataDog/agent-integrations
+/varnish                            @DataDog/agent-integrations
+/voltdb                             @DataDog/agent-integrations
+
+# Database monitoring
+/sqlserver                          @DataDog/database-monitoring
+
+# Infrastructure integrations
+/snmp                               @DataDog/infrastructure-integrations
```
Comment (FlorentClarret) [inline .github/CODEOWNERS:22]:
```suggestion
/snmp                               @DataDog/network-device-monitoring
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+/openldap                           @DataDog/agent-integrations
+/pgbouncer                          @DataDog/agent-integrations
+/powerdns_recursor                  @DataDog/agent-integrations
+/varnish                            @DataDog/agent-integrations
+/voltdb                             @DataDog/agent-integrations
+
+# Database monitoring
+/sqlserver                          @DataDog/database-monitoring
+
+# Infrastructure integrations
```
Comment (FlorentClarret) [inline .github/CODEOWNERS:21]:
```suggestion
# Network Device Monitoring
```

**Review by hithwen — APPROVED** [review]

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+# Agent integrations
+/cacti                              @DataDog/agent-integrations
+/druid                              @DataDog/agent-integrations
+/ibmmq                              @DataDog/agent-integrations
+/kyototycoon                        @DataDog/agent-integrations
+/memcached                          @DataDog/agent-integrations
+/nginx                              @DataDog/agent-integrations
+/openldap                           @DataDog/agent-integrations
+/pgbouncer                          @DataDog/agent-integrations
+/powerdns_recursor                  @DataDog/agent-integrations
```
Comment (hithwen) [inline .github/CODEOWNERS:14]:
```suggestion
/powerdns_recursor                  @DataDog/agent-integrations
/supervisord                        @DataDog/agent-integrations
```

Comment (FlorentClarret) [reply .github/CODEOWNERS:14]: Thank you!

**Review by pducolin — APPROVED** [review]
Ok for @DataDog/network-device-monitoring

**Review by pducolin — APPROVED** [review]

**Review by AlexandreYang — APPROVED** [review]
Good for Network Device Monitoring

**Review by clamoriniere — APPROVED** [review]

**Review by jmeunier28 — APPROVED** [review]

**Pull request merged**